lib/manager/dockerfile/extract.ts

```diff
@@ -7,6 +7,24 @@ import type { PackageDependency, PackageFile } from '../types';
 
 export function splitImageParts(currentFrom: string): PackageDependency {
   if (currentFrom.includes('$')) {
+    // Check if the variable contains default value, and if so return
+    // it as a valid dependency
+    const variableRegex = /^\$\{(.+):-(.+):(.+)@sha256:(.+)\}$/i;
```

Comment:
Tag and / or digest should be optional

Reply:
Hi @viceice could you please explain why tag/digest *should* be optional? I guess they *can* be optional, but in this issue, I need to return a valid dependency only if the variable has the format of `${REDIS_IMAGE:-redis:5.0.0@sha256:abcd}` (I mean I only process variables when they provide enough info for dependency update). To be honest, I'm confused here since I'm new to this topic. 

Could you please advice how the variable looks like if the tag/digest is optional? For example, does it look like `${REDIS_IMAGE:-redis:5.0.0` for missing digest, and `${REDIS_IMAGE:-redis@sha256:abcd}` for missing hash?

Reply:
Yea that's how it looks